### DISCUSSION TITLE
Long press in bankable button

### DISCUSSION THREAD

**Discussion by MiOiM** [tttapa/Control-Surface#642] [Q&A]
Is there a way to have the ReleasedShort/IncrementLong functionality in a bankable button?

IncrementButton doesn't seem to be bankable and I don't see these methods in CCIncrementDecrementButtons...

Thanks!

**Comment by tttapa** [comment]
CCIncrementDecrementButtons already uses ReleasedShort and IncrementLong, but if you want custom behavior, I'd recommend following the [Custom-MIDI-Output-Element-Bankable](https://tttapa.github.io/Control-Surface-doc/Doxygen/d4/d6b/Custom-MIDI-Output-Element-Bankable_8ino-example.html) example. Note that `IncrementButton` is concerned only with reading the state of a push button, it doesn't send MIDI on its own and has no concept of banks.
